test/Swashbuckle.AspNetCore.Newtonsoft.Test/SchemaGenerator/NewtonsoftSchemaGeneratorTests.cs

```diff
@@ -142,6 +142,33 @@ public void GenerateSchema_GeneratesObjectSchema_IfDictionaryTypeHasEnumKey()
             Assert.Equal(new[] { "Value2", "Value4", "Value8" }, schema.Properties.Keys);
         }
 
+        [Fact]
+        public void GenerateSchema_GeneratesObjectSchema_IfDictionaryTypeHasEnumKey_CamelCasePropertyNamesContractResolver()
+        {
+            JsonSerializerSettings settings = new JsonSerializerSettings()
+            {
+                ContractResolver = new CamelCasePropertyNamesContractResolver()
+            };
+
+            var schema = Subject(null, (s) =>
+            {
+                s.ContractResolver = new CamelCasePropertyNamesContractResolver();
+            }).GenerateSchema(typeof(IDictionary<IntEnum, int>), new SchemaRepository());
+
+            Assert.Equal("object", schema.Type);
+            Assert.Equal(new[] { "value2", "value4", "value8" }, schema.Properties.Keys);
+
+            var a = new NewtonsoftDataContractResolver(settings);
+            var b = a.GetDataContractForType(typeof(IDictionary<IntEnum, int>)).JsonConverter(new Dictionary<IntEnum, int>()
+            {
+                { IntEnum.Value2, 2 }, { IntEnum.Value4, 4 }, { IntEnum.Value8, 8 },
+            });
+            foreach (var key in schema.Properties.Keys)
+            {
+                Assert.Contains(key, b);
+            }
```

Comment:
This feels out of place here, so I'd welcome pointers on where such a test should live. I was hoping to find a place in IntegrationTests, but I didn't see an obvious one. It also only checks that the key is present somewhere in the serialized string, but maybe that's good enough.

Reply:
You could add a resource to one of the test websites that does some serialization with this configuration, and then add a verification test. There should be _somewhere_ sensible were we already have verification tests that this can live.

Reply:
I think this part is not that important, so I removed it instead